lib/Elastica/Response.php

```diff
@@ -133,13 +126,7 @@ public function getFullError()
      */
     public function getErrorMessage()
     {
-        $error = $this->getError();
-
-        if (!is_string($error)) {
```

Comment:
Not sure if we introduced this to make master "partially" compatible with 2.x. Happy to remove it now and lets readd it in case things break ...